src/roadmap/common.py

```
@@ -79,33 +73,17 @@ async def query_rbac(
         return [{}]
 
     url = f"{settings.rbac_url}/api/rbac/v1/access/?{urllib.parse.urlencode(params, doseq=True)}"
-    logger.debug(
-        "RBAC request: url=%s identity_present=%s identity_type=%s",
-        url,
-        "X-RH-Identity" in headers,
-        type(x_rh_identity).__name__,
-    )
 
     def _fetch_rbac():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_fetch_rbac`

(ANN202)